.node-version

```diff
@@ -0,0 +1 @@
+18
```

Comment:
Is it important to use exactly v18?

Currently, `wrangler` can't build worker due to different versions (wrangler's action is under node 16.x)